src/components/DraggableList/useDraggableInPortal.ts

```diff
@@ -0,0 +1,43 @@
+import type {DraggableChildrenFn} from 'react-beautiful-dnd';
+import {useEffect, useRef} from 'react';
+import {createPortal} from 'react-dom';
+
+type DraggableInPortalProps = {
+    shouldUsePortal?: boolean;
+};
+
+export default function useDraggableInPortal({shouldUsePortal}: DraggableInPortalProps): (render: DraggableChildrenFn) => DraggableChildrenFn {
+    const element = useRef<HTMLDivElement>(document.createElement('div')).current;
+
+    useEffect(() => {
+        if (!shouldUsePortal || !element) {
+            return () => {};
+        }
+        element.style.pointerEvents = 'none';
+        element.style.position = 'absolute';
+        element.style.height = '100%';
+        element.style.width = '100%';
+        element.style.top = '0';
+
+        document.body.appendChild(element);
+
+        return () => {
+            document.body.removeChild(element);
+        };
+    }, [element, shouldUsePortal]);
+
+    if (!shouldUsePortal) {
+        return (render) => render;
+    }
+
+    return (render) => (provided, snapshot, rubric) => {
+        const result = render(provided, snapshot, rubric);
+        const style = provided.draggableProps.style;
+        if (style && 'position' in style && style.position === 'fixed') {
+            return createPortal(result, element);
+        }
+        return result;
```

Comment:
I got a type error for `result`. Can you check if you also get the same error?

<img width="400" alt="Screenshot 2023-10-03 at 11 25 09 AM" src="https://github.com/Expensify/App/assets/98560306/61c5e6df-c277-4f98-91e4-cb7a25458bff">

Reply:
okay maybe my dev setup then thank you for confirming